playground/static/main.js

```diff
@@ -238,6 +238,8 @@ async function updateOutput(input) {
                 renderIframe.style.display = "none";
                 render.style.display = "none";
 
```

Comment:
Maybe remove this newline to improve style, so that all "set style"-statements stick together